### ISSUE TITLE
Accessibility: No label on the Date field in /events view

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_events#88]
![Image](https://user-images.githubusercontent.com/3852805/205015222-b6d20b6d-cb4f-4b89-adf0-89abdc64514f.png)

**Comment by msayoung** [comment]
https://github.com/localgovdrupal/localgov_events/pull/89
